src/styles/reset.scss

```diff
@@ -0,0 +1,7 @@
+body,
+p,
+ul,
+h2 {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
All of this element's don't have any default padding

Reply:
I found the information that ul have default padding-left: 40px
It would be better to write it separately from other resets like now or together?
![image](https://user-images.githubusercontent.com/110456763/186158480-1d51fc0a-0e61-4a31-a36b-46f66677e1d9.png)

Reply:
Yeah, I was wrong about ul, sorry. Better to write it separately